x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/Driver.java

```diff
@@ -49,6 +49,22 @@ public class Driver implements Releasable, Describable {
     public static final TimeValue DEFAULT_STATUS_INTERVAL = TimeValue.timeValueSeconds(1);
 
     private final String sessionId;
+
+    /**
+     * The wall clock time when this driver was created in milliseconds since epoch.
+     * Compared to {@link #startNanos} this is less accurate and is measured by a
+     * timer that can go backwards. This is only useful for presenting times to a
+     * user, like over the status API.
+     */
+    private final long startTime;
```

Comment:
Never mind, makes sense to have the absolute starting time.

Reply:
Yeah - we need an epoch time so I just called it `time` instead of `nanos`. It's not really clear from the names that `nanos` means "starts and some arbitrary point in time, not epoch" but that *is* the only way you can get a nanosecond precision timer.